.idea/compiler.xml

```diff
@@ -2,8 +2,8 @@
 <project version="4">
   <component name="CompilerConfiguration">
     <bytecodeTargetLevel>
-      <module name="mvc-auth-commons_main" target="1.7" />
-      <module name="mvc-auth-commons_test" target="1.8" />
+      <module name="mvc-auth-commons_main" target="1.8" />
```

Comment:
for some reason, this folder is checked into the repository. Once I added the toolchain block, this file got updated.